Dockerfile.build

```diff
@@ -0,0 +1,3 @@
+FROM maven:3-jdk-7-onbuild
+
+CMD ["mvn test"]
```

Comment:
As I explained before this is for a docker based build and development environment.
If you don't think it's useful I'll remove it.